frameworks/hdfs/docs/limitations.md

```diff
@@ -28,4 +28,8 @@ When the service is deployed on a virtual network, the service may not be switch
 
 ## Task Environment Variables
 
-Each service task has some number of environment variables, which are used to configure the task. These environment variables are set by the service scheduler. While it is _possible_ to use these environment variables in adhoc scripts (e.g. via `dcos task exec`), the name of a given environment variable may change between versions of a service and should not be considered a public API of the service.
\ No newline at end of file
+Each service task has some number of environment variables, which are used to configure the task. These environment variables are set by the service scheduler. While it is _possible_ to use these environment variables in adhoc scripts (e.g. via `dcos task exec`), the name of a given environment variable may change between versions of a service and should not be considered a public API of the service.
+
+## Zones
```

Comment:
Should we use `racks` in parentheses since that's HDFS-specific terminology?

Reply:
We should decide on some uniform language that's probably along the lines of:

"DC/OS Zones allow the <blank> service to implement <blank specific terminology>